cbox/cliparser.py

```diff
@@ -13,29 +13,48 @@
 __all__ = ('get_cli_parser', 'parse_args', )
 
 
+class Arg:
+    def __init__(
+            self, type_=str, positional=False, names=None, help_=None):
+        self.names = names
+        self.type_ = type_
+        self.positional = positional
+        self.help_ = help_
+
+    def get_names(self):
+        if self.positional:
+            return self.names
+
+        return list(map(lambda _: ('--' if len(_) > 1 else '-') + _,
```

Comment:
I think naming the lambda will make it easier to read and modify (these things tend to get more complex).

Maybe `prefix_flag`?